FreeRTOS/FreeRTOS
# Uncrustify PR Check Changes

Uncrustify PR Check Changes

Description
-----------
Put the uncrustify parsing in a log group
Print out a git diff of the changes if a file is incorrectly formatted so end user can see it

Test Steps
-----------
Run the CI-Checker GitHub Action
[Working Run One](https://github.com/Skptak/FreeRTOS/actions/runs/5549759684/jobs/10134187166#step:4:1), [Working Run Two](https://github.com/Skptak/FreeRTOS/actions/runs/5549811029/jobs/10134289515#step:4:1), Forced a failure on [this run](https://github.com/Skptak/FreeRTOS/actions/runs/5549825239/jobs/10134319642#step:4:290) to show how it looks now

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
